lib/philips.js

```diff
@@ -249,4 +659,9 @@ function encodeGradientColors(value, opts) {
 module.exports = {
     decodeGradientColors,
     encodeGradientColors,
+    hueExtend,
```

Comment:
Since this is in `philips.js`, we can rename this to just `extend`. (we do the same for TuYa)